<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(22, 23, 24);
background-color: rgb(30, 30, 30);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="PlotWidget" name="trace"/>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="Line" name="line">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(50, 50, 51);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="PlotWidget" name="hist"/>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string>Trace Settings</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="2" column="1">
            <widget class="QLineEdit" name="traceAxisLine">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(241, 241, 241);
color: rgb(0, 0, 0);</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 252, 246)</string>
             </property>
             <property name="text">
              <string>Trace Height</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 252, 246)</string>
             </property>
             <property name="text">
              <string>Trace Period (ms)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="tracePeriodLine">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(241, 241, 241);
color: rgb(0, 0, 0);</string>
             </property>
             <property name="maxLength">
              <number>4</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 252, 246);</string>
             </property>
             <property name="text">
              <string>Trace Bin Size (ms)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="traceBinCombo">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(241, 241, 241);
color: rgb(0, 0, 0);</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Coarse Trace Toggle</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QPushButton" name="coarseTraceButton">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(241, 241, 241);
color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>OFF</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="coarseTraceLine">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(241, 241, 241);
color: rgb(0, 0, 0);</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Coarse Trace Time (s)</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Reset Coarse Trace</string>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QPushButton" name="resetCoarseButton">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(241, 241, 241);
color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_6">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 252, 246);</string>
             </property>
             <property name="text">
              <string>FRET Toggle</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QPushButton" name="fretButton">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(241, 241, 241);
color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>OFF</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Coarse Trace Auto-Scale</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QPushButton" name="coarseScaleButton">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(241, 241, 241);
color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>ON</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string>Histogram Settings</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 252, 246);</string>
             </property>
             <property name="text">
              <string>Hist Height</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="histAxisLine">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(241, 241, 241);
color: rgb(0, 0, 0)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 252, 246);</string>
             </property>
             <property name="text">
              <string>Hist Bin Size (ps)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="histBinCombo">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(241, 241, 241);
color: rgb(0, 0, 0);</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_7">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 252, 246);
</string>
             </property>
             <property name="text">
              <string>File Directory</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="fileButton">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(241, 241, 241);
color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>Select File</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>54</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
